### ISSUE TITLE
How to reset autoLoadJavascriptFile() - Android

### ISSUE THREAD

**Issue by erjdriver** [Notalib/nativescript-webview-ext#74]
For a WebViewExt - I've set autoLoadJavascriptFile().

Sometimes - in the middle of the session the Javascript code can change - if I call autoLoad again - 2 copies of the code (old and new get loaded).

How can I - delete/replace previous autoload file?

I hope the file contents are not cached somewhere - because I change the contents of the same file and would like the new contents to be injected in subsequent page loads.
I tried the following before the 2nd call to autoLoad - but didn't work

`this.webview.unregisterLocalResource( "js_code");`

**Comment by m-abs** [comment]
You should be able to reset it with `removeAutoLoadJavaScriptFile("js_code")`

**Comment by erjdriver** [comment]
My apologies.

I did look up on your main doc page and that method wasn't there.

This method almost solves the issue - it happens only once after I remove and set the autoload file - why am I doing this - because the contents of the js file changes often. Why am I not using `loadJavascriptfile` because as you know - that has an issue with https on ios.

Anyway I remove and set the new autoload file - and then the first thing I do is

```
       this.webview.loadUrl( "about:blank" ).then( (value) =>
       {
           this.webview.loadUrl( MY_URL );
       }
```

I get the following error message - just once after about:blank page is loaded.

I do the about:blank because sometimes the webview is already at MY_URL and it doesn't trigger the `loaded` event. so forcing to a different page.

`chromium: [INFO:CONSOLE(1)] "Uncaught SyntaxError: Identifier 'ERROR_PREFIX' has already been declared", source: x-local://js_code (1)`

ERROR_PREFIX is a global variable in my js file.

I only get it once per session - and at this stage - I do several loads after this.

Maybe this page was already loaded w/ my js code and it was re-inserted?


Previously I was getting this message for all my loads and also multiple times depending on the number of times I ran the session (3x if I ran session 3x) - so definitely the major problem is solved.
